torch/csrc/jit/passes/quantization.cpp

```diff
@@ -676,5 +677,252 @@ TORCH_API script::Module InsertObservers(
   return input_module;
 }
 
+Node* insertQuantDeQuantCall(Value* v, const IValue& qparams, at::ScalarType t, bool insert_after=true) {
+  Graph* g = v->node()->owningGraph();
+  Node* quant = createQuantNode(v, g);
+  Node* intrepr = createIntReprNode(v, g);
+  Node* dequant = createDeQuantNode(v, g);
+  Node* insert_point = insert_after ? v->node() : *g->nodes().begin();
+  WithCurrentScope scope_guard(
+      *insert_point->owningGraph(), insert_point->scope());
+  WithInsertPoint ins(insert_point);
+
+  // Add quant-intrepr-dequant nodes and replace for all uses of Value
+  // Create qparam constant nodes
+  TORCH_INTERNAL_ASSERT(qparams.isTuple(), "qparams must be tuple");
+  auto tp = qparams.toTuple();
+  IValue scale = tp->elements()[0].toTensor().item().toFloat();
+  IValue zero_point = tp->elements()[1].toTensor().item().toInt();
+  Value* scale_val = g->insertConstant(scale);
+  Value* zero_point_val = g->insertConstant(zero_point);
+
+  // Insert quant/int_repr/dequant nodes
+  if (insert_after) {
+    quant->insertAfter(insert_point);
+  } else {
+    quant->insertBefore(insert_point);
+  }
+
+  intrepr->insertAfter(quant);
+  dequant->insertAfter(intrepr);
+
+  // Attach inputs to quantization pattern nodes
+  quant->addInput(v);
+  intrepr->addInput(quant->output());
+  dequant->addInput(intrepr->output());
+
+  quant->addInput(scale_val);
+  quant->addInput(zero_point_val);
+  dequant->addInput(scale_val);
+  dequant->addInput(zero_point_val);
+
+  Value* scalar_type_val = insertScalarType(quant, t);
+  TORCH_INTERNAL_ASSERT(scalar_type_val != nullptr);
+  quant->addInput(scalar_type_val);
+  dequant->addInput(scalar_type_val);
+  return dequant;
+}
+
+std::tuple<std::string, c10::optional<script::Module>> findObserverModule(
```

Comment:
This function have very non-trivial signature. It at least needs to be commented, but might make sense to rethink what it does and returns - I wouldn't expect a `findObject` function to return some string along with the object. It might make sense to split it into to functions if you need to perform two actions (one of which is sometimes ignored as well).